assets/src/constants.js

```diff
@@ -105,6 +105,9 @@ export const BLOCK_TAG_MAPPING = {
 	'core/embed': 'figure',
 	'core/image': '.wp-block-image',
 	'amp/amp-story-text': 'p,h1,h2',
+	'amp/amp-story-post-author': 'p,h1,h2',
+	'amp/amp-story-post-date': 'p,h1,h2',
+	'amp/amp-story-post-title': 'p,h1,h2',
```

Comment:
`amp/amp-story-post-title` is `h1` only. It doesn't change tags like `amp/amp-story-text`.

Similarly, `amp/amp-story-post-date` and `amp/amp-story-post-author` just use a `div`.

Reply:
Oh, that's right, thanks. Fixed within 1ca5831.